### DISCUSSION TITLE
Docker Compose - Error on adding Community Container s

### DISCUSSION THREAD

**Discussion by ebildebil** [nextcloud/all-in-one#3577] [Questions] [answered]
I tried to add the local AI container today by adding the below line in my docker-compose.yml file:
- AIO_COMMUNITY_CONTAINERS=local-ai

When restarting the AIO container i get the below error:
You've set AIO_COMMUNITY_CONTAINERS but at least one container was not found.
It is set to 'local-ai'.
The community container local-ai was not found!
You've set AIO_COMMUNITY_CONTAINERS but at least one container was not found.
It is set to 'local-ai'.

Any idea what I am doing wrong? I've tried with quotes and without.

**Comment by ebildebil** [comment, marked as answer]
Just read #3561, probably because I am on the stable channel.

**Reply by szaimen** [reply]
This is now released with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by ebildebil** [reply]
I managed to install 2 community containers today:
Plex and Local-AI.

I have some issues with them, I'll start separate discussions for them. Thank you.
